src/components/rocket/RocketsBookingButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { bookingRocket, bookingCancel } from '../../redux/rockets/rocketsSlice';
+
+const RocketsBookingButton = (props) => {
+  const { id, reserved } = props;
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      {
+        reserved
+          ? <button id={id} type="button" className="btn btn-light" onClick={() => dispatch(bookingCancel(id))}>Cancel Reservation</button>
+          : <button type="button" className="btn btn-primary" onClick={() => dispatch(bookingRocket(id))}>Reserve Rocket</button>
+      }
```

Comment:
-  Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
You can use the syntax:
`{ reserved && ( // code for reserved rocket)}
{ !reserved && ( // code for not reserved rocket)}`